posts/2024-10-31-Project-Goals-Oct-Update.md

```diff
@@ -0,0 +1,271 @@
+
+Finally, we have been finding an increasing number of stabilization requests at the compiler level, and so @wesleywiser and @davidtwco from the compiler team have started attending meetings to create a faster response. One of the results of that collaboration is [RFC #3716], authored by Alice Ryhl, which proposes a method to manage compiler flags that modify the target ABI. Our previous approach has been to create distinct targets for each combination of flags, but the number of flags needed by the kernel make that impractical. Authoring the RFC revealed more such flags than previously recognized, including those that modify LLVM behavior.
+
+[RFC #3716]: https://github.com/rust-lang/rfcs/pull/3716
+
+<div style="display: flex;" class="mt2 mb3">
+    <div style="flex: auto;"><a href='https://github.com/rust-lang/rust-project-goals/issues/117'><strong>Rust 2024 Edition</strong></a></div>
+    <div style="flex: initial;"><progress value="16" max="30"></progress>
+</div>
+</div>
+
+ The Rust 2024 edition is progressing well and is on track to be released on schedule. The major milestones include preparing to stabilize the edition by November 22, 2024, with the actual stabilization occurring on November 28, 2024. The edition will then be cut to beta on January 3, 2025, followed by an announcement on January 9, 2025, indicating that Rust 2024 is pending release. The final release is scheduled for February 20, 2025.
+
+The priorities for this edition have been to ensure its success without requiring excessive effort from any individual. The team is pleased with the progress, noting that this edition will be the largest since Rust 2015, introducing many new and exciting features. The process has been carefully managed to maintain high standards without the need for high-stress heroics that were common in past editions. Notably, the team has managed to avoid cutting many items from the edition late in the development process, which helps prevent wasted work and burnout.
+
+All priority language items for Rust 2024 have been completed and are ready for release. These include several key issues and enhancements. Additionally, there are three changes to the standard library, two updates to Cargo, and an exciting improvement to `rustdoc` that will significantly speed up doctests.
+
+This edition also introduces a new style edition for `rustfmt`, which includes several formatting changes. The only remaining task is the completion of a PR for one item, after which final quality assurance crater runs will be conducted. Once these are triaged, the nightly beta for Rust 2024 will be announced, and wider testing will be solicited.
```

Comment:
```suggestion
This edition also introduces a new style edition for `rustfmt`, which includes several formatting changes.

The team is preparing to start final quality assurance crater runs. Once these are triaged, the nightly beta for Rust 2024 will be announced, and wider testing will be solicited.
```

The situation has now changed.  Everything we were waiting for has now happened.